### ISSUE TITLE
Warnings when importing pyemma

### ISSUE THREAD

**Issue by stefdoerr** [markovmodel/PyEMMA#683]
Are you guys also seeing these mountains of warnings when importing pyemma?
It has been happening a month now since when I ran a conda update.
I tried a fresh install of miniconda and anaconda on different machines and I am seeing it everywhere.

``` python
In [1]: import pyemma                                                                                                                                                                                            
/shared/sdoerr/Software/anaconda3/lib/python3.4/site-packages/jupyter_core/application.py:106: DeprecationWarning: metadata {'config': True} was set from the constructor.  Metadata should be set using the .tag() method, e.g., Int().tag(key1='value1', key2='value2')
  help="""Generate default config file."""
/shared/sdoerr/Software/anaconda3/lib/python3.4/site-packages/jupyter_core/application.py:110: DeprecationWarning: metadata {'config': True} was set from the constructor.  Metadata should be set using the .tag() method, e.g., Int().tag(key1='value1', key2='value2')
  help="Specify a config file to load."
/shared/sdoerr/Software/anaconda3/lib/python3.4/site-packages/jupyter_core/application.py:118: DeprecationWarning: metadata {'config': True} was set from the constructor.  Metadata should be set using the .tag() method, e.g., Int().tag(key1='value1', key2='value2')
  help="""Full path of a config file.""",
```

etc etc.

I have to load pyemma at the beginning of my scripts and then disable the warnings like this:

``` python
import pyemma
import warnings
warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Otherwise the notebooks hang from too much warning printing when I used for example TICA.

I guess I will just head over to the jupyter github page and ask around. https://github.com/jupyter/jupyter_core/issues/71
Reporting it here so you know that your other users are probably affected as well.

**Comment by franknoe** [comment]
I wasn't aware of this. Are you using the latest release 2.0.3? I don't 
have these warnings, but I am working with the devel version.

Am 04/02/16 um 17:38 schrieb Stefan:

> Are you guys also seeing these mountains of warnings when importing 
> pyemma?
> I know they are not directly related to pyemma, just wondering if you 
> have reported them.
> It has been happening a month now since when I ran a conda update.
> I tried a fresh install of miniconda and anaconda on different 
> machines and I am seeing it everywhere.
> 
> In [1]:import  pyemma
> /shared/sdoerr/Software/anaconda3/lib/python3.4/site-packages/jupyter_core/application.py:106:DeprecationWarning: metadata {'config':True} wasset  from  the constructor.  Metadata should beset  using the .tag() method, e.g., Int().tag(key1='value1',key2='value 2') help="""Generate default config file.""" 
> /shared/sdoerr/Software/anaconda3/lib/python3.4/site-packages/jupyter_core/application.py:110: 
> DeprecationWarning: metadata {'config': True} was set from the 
> constructor. Metadata should be set using the .tag() method, e.g., 
> Int().tag(key1='value1', key2='value 2') help="Specify a config file 
> to load." 
> /shared/sdoerr/Software/anaconda3/lib/python3.4/site-packages/jupyter_core/application.py:118: 
> DeprecationWarning: metadata {'config': True} was set from the 
> constructor. Metadata should be set using the .tag() method, e.g., 
> Int().tag(key1='value1', key2='value 2') help="""Full path of a config 
> file.""",
> 
> etc etc.
> 
> I have to load pyemma at the beginning of my scripts and then disable 
> the warnings like this:
> 
> import  pyemma
> import  warnings
> warnings.filterwarnings("ignore",category=DeprecationWarning)
> 
> Otherwise the notebooks hang from too much warning printing when I 
> used for example TICA.
> 
> Should we head over to the jupyter github page and make some noise? :P
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/markovmodel/PyEMMA/issues/683.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

**Comment by stefdoerr** [comment]
They commented on the issue that it is on their side (as can be seen in the warnings). Latest pyemma version available in conda together with latest jupyter version causes it. I will try to fix it with them.

**Comment by franknoe** [comment]
Ah I see. Thanks. Anyway, we're happy to see if we can work around it, 
if it appears to be a more complex issue.

Am 04/02/16 um 19:42 schrieb Stefan:

> They commented on the other issue that it is on their side (as can be 
> seen in the warnings). Latest pyemma version available in conda 
> together with latest jupyter version causes it. I will try to fix it 
> with them.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/markovmodel/PyEMMA/issues/683#issuecomment-179993986.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

**Comment by stefdoerr** [comment]
Turns out it might be a problem coming from pyemma after all. Take a look at the issue thread 
https://github.com/jupyter/jupyter_core/issues/71

**Comment by franknoe** [comment]
So apparently there are multiple sources. I think @marscher or @clonker 
need to look at this next week, they are currently both traveling so 
this will have to wait til monday or tuesday. Thanks for pointing it out 
Stefan.

Am 05/02/16 um 11:28 schrieb Stefan:

> Turns out it might be a problem coming from pyemma after all. Take a 
> look at the issue thread
> jupyter/jupyter_core#71 
> https://github.com/jupyter/jupyter_core/issues/71
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/markovmodel/PyEMMA/issues/683#issuecomment-180286703.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

**Comment by franknoe** [comment]
@clonker provided a fix. The latest devel versions of PyEMMA and mdtraj (you need both) should behave well. The former has already been released as v2.0.4, while the pip version will be released tomorrow. Please have a try.
